docs/build-parachains-rococo.md

```diff
@@ -141,6 +141,21 @@ process are the following:
 2. Maintain at least one parachain collator
 3. Sign-up through the [Rococo V1 Parachain Registration](https://forms.gle/Eacp7RaRm3VNion16) form
 
+*Update: 15. Jan. 2021*
+
+- If you are considered to be included, the Rococo team will get in touch with you through the Element handle provided in the sign-up form, a rough estimate of going forward will be (in no certain order within the batch and please be assure of 5., 6., 7.):
```

Comment:
"be assure of" sounds wrong, and also the items below are not numbered, maybe number them to match this?